docs/publish/centralized-deployment.md

```diff
@@ -13,15 +13,15 @@ The Office 365 admin center makes it easy for an administrator to deploy Office
 The Office 365 admin center currently supports the following scenarios:
 
 - Centralized Deployment of new and updated add-ins to individuals, groups, or an organization.
-- Deployment to multiple platforms, including Windows and Office Online, with Mac coming soon.
+- Deployment to multiple platforms, including Windows, Mac, and Office Online.
 - Deployment to English language and worldwide tenants.
 - Deployment of cloud-hosted add-ins.
 - Deployment of add-ins that are hosted within a firewall.
 - Deployment of AppSource add-ins.
 - Automatic installation of an add-in for users when they launch the Office application.
 - Automatic removal of an add-in for users if the admin turns off or deletes the add-in, or if users are removed from Azure Active Directory or from a group to which the add-in has been deployed.
 
-Centralized Deployment is the recommended way for an Office 365 admin to deploy Office Add-ins within an organization, provided that the organization meets all requirements for using Centralized Deployment. For information about how to determine if your organization can use Centralized Deployment, see [Determine if Centralized Deployment of add-ins works for your Office 365 organization](https://support.office.com/article/Determine-if-Centralized-Deployment-of-add-ins-works-for-your-Office-365-organization-B4527D49-4073-4B43-8274-31B7A3166F92).
+Centralized Deployment is the recommended way for an Office 365 admin to deploy Office Add-ins within an organization, provided that the organization meets all requirements for using Centralized Deployment. For information about how to determine if your organization can use Centralized Deployment, see [Determine if Centralized Deployment of add-ins works for your Office 365 organization](/office365/admin/manage/centralized-deployment-of-add-ins).
```

Comment:
This link doesn't work on the staging site. I think it should still be the link to the support article.

Reply:
@AlexJerabek: If you try the support link, it redirects to this (https://docs.microsoft.com/en-us/office365/admin/manage/centralized-deployment-of-add-ins?redirectSourcePath=%252farticle%252fDetermine-if-Centralized-Deployment-of-add-ins-works-for-your-Office-365-organization-B4527D49-4073-4B43-8274-31B7A3166F92&view=o365-worldwide) so I cut out the middle man. From the staging site, you may need to switch to master branch or go to PROD site.

Reply:
Okay. For some reason I didn't think we could relative links outside of Office Add-ins content.